Lesson 12/src/index.js

```diff
@@ -0,0 +1,108 @@
+// 1
+
+let number = 0
+
+while (number <= 20) {
+  console.log(number)
+  if (number === 15) {
+    break
+  }
+  number += 1
+  // if (a === 10) {
+  //   continue
+  // }
+}
+
+// 2
+function findEvenNumber(number1) {
+  if (number1 % 2 === 0) {
+    return true
+  } return false
+}
+findEvenNumber(16)
+findEvenNumber(7)
+
+// 3
+
+const users = [
+  { name: 'Alice', age: 25 },
+  { name: 'Bob', age: 30 },
+  { name: 'Charlie', age: 35 }
+]
+
+users.forEach((element) => {
+  console.log(`Имя пользователя: ${element.name}, Возраст: 
+    ${element.age}`)
```

Comment:
```suggestion
  console.log(`Имя пользователя: ${element.name}, Возраст: ${element.age}`)
```

не используй переносы в строке, они будут видны в результате вывода:

<img width="426" alt="image" src="https://github.com/user-attachments/assets/5feb919c-cee9-4e05-97b1-7616231d3609">